pulp_rpm/tests/functional/constants.py

```diff
@@ -232,3 +232,8 @@
 RPM_KICKSTART_FIXTURE_SUMMARY = {
     RPM_KICKSTART_CONTENT_NAME: RPM_KICKSTART_COUNT
 }
+
+
+CENTOS7_URL = "http://mirror.linux.duke.edu/pub/centos/7/os/x86_64/"
+CENTOS8_APPSTREAM_URL = "http://mirror.linux.duke.edu/pub/centos/8/AppStream/x86_64/kickstart/"
```

Comment:
What is the reason for choosing a kickstart repo and not the usual/normal one?
Same question for the BaseOS url.

Reply:
no reason, I had to pick a URL, so I picked those

Reply:
Let's make this one a normal one http://mirror.linux.duke.edu/pub/centos/8/AppStream/x86_64/os/
And keep one kickstart `CENTOS8_BASEOS_URL = "http://mirror.linux.duke.edu/pub/centos/8/BaseOS/x86_64/kickstart/"`
Please rename it to CENTOS8_KICKSTART_BASEOS_URL so it was clear that it's not a usual repo

Reply:
Oh there are already kickstatrs in these tests. Then I suggest to have a normal centos8 repo